sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb

```diff
@@ -569,16 +591,114 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create serverless endpoint - names must be unique, we will use suffix of the model\n",
+    "short_id = registered_model_name[-9:]\n",
+    "serverless_endpoint_name = \"my-endpoint-\" + short_id\n",
+    "\n",
+    "serverless_endpoint = ServerlessEndpoint(\n",
+    "    name=serverless_endpoint_name,\n",
+    "    model_id=rg_model_asset_id,\n",
+    ")\n",
+    "\n",
+    "created_endpoint = ml_client.serverless_endpoints.begin_create_or_update(\n",
+    "    serverless_endpoint\n",
+    ").result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sample inference against the deployed endpoint (optional)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = created_endpoint.scoring_uri\n",
+    "key = ml_client.serverless_endpoints.get_keys(created_endpoint.name).primary_key\n",
+    "model = ChatCompletionsClient(\n",
+    "    endpoint=url,\n",
+    "    credential=AzureKeyCredential(key),\n",
+    ")\n",
     "\n",
-    "Once the above job completes, you should be able to deploy the model and use it for inferencing. To deploy this model, do the following:\n",
+    "response = model.complete(\n",
+    "    messages=[\n",
+    "        SystemMessage(\n",
+    "            content=\"You are a helpful assistant. Your output should only be one of the five choices: 'A', 'B', 'C', 'D', or 'E'.\"\n",
+    "        ),\n",
+    "        UserMessage(\n",
+    "            content=\"Answer the following multiple-choice question by selecting the correct option.\\n\\nQuestion: Can you name a good reason for attending school?\\nAnswer Choices:\\n(A) get smart\\n(B) boredom\\n(C) colds and flu\\n(D) taking tests\\n(E) spend time\"\n",
+    "        ),\n",
+    "    ],\n",
+    ")\n",
     "\n",
-    "* Go to AI Studio\n",
-    "* Navigate to the Fine-tuning tab on the left menu\n",
-    "* In the list of models you see, click on the model which got created from the distillation\n",
-    "* This should take you to the details page where you can see the model attributes and other details\n",
-    "* Click on the Deploy button on top of the page\n",
-    "* Follow the steps to deploy the model"
+    "print(response.choices[0].message.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cleanup endpoints created (optional)\n",
+    "\n",
+    "Endpoint deployments are chargeable and incurr costs on the subscription. Optionally clean up the endpoints after finishing experiments"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# _ = ml_client.serverless_endpoints.begin_delete(TEACHER_MODEL_ENDPOINT_NAME)\n",
+    "# _ = ml_client.serverless_endpoints.begin_delete(serverless_endpoint_name)"
```

Comment:
Might want to keep the deletion code but comment out.

Reply:
Done.